windows_containers/enabling-windows-container-workloads.adoc

```diff
@@ -26,6 +26,11 @@ Dual NIC is not supported on WMCO-managed Windows instances.
 
 * You are running an {product-title} cluster version 4.6.8 or later.
 
+[NOTE]
+====
+The WMCO is not supported in clusters that use a xref:../networking/enable-cluster-wide-proxy.html[cluster-wide proxy] because the WMCO is not able to route traffic through the proxy connection for the workloads.
```

Comment:
I would suggest not using a definite article with Operator abbreviations. So, "WMCO is not supported...", not "the WMCO is not supported...". This probably comes down to preference. Search for "product name definite article" in the CCS gChat for further discussion.

Reply:
@aireilly Thanks. Do you have a link to that convo? I cannot seen to find it. (I am not much of a gChat user.)
If you don't mind, that sounds like a larger change, as there are many _The WMCO_ references in the docs.